### PR TITLE
Added Side Menu Bar and Drawer

### LINKED ISSUE #295: Create components for the Side-menu bar and Footer-text.
As per the following designs, create components for the Side-menu bar and Footer-text.

Screenshot (908)

Reference for the design: https://www.figma.com/file/65I2EqaTgwUnYHUeWLfvJr/components?node-id=0%3A1

### PR DESCRIPTION
**Pull request by Janvi-Thakkar**
## Description
Side menuBar would appear for screen-width>750 and Drawer MenuBar would appear for screen-width<=750

## Related Issue
Fixes #295 

## Screenshots or GIF (In case of UI changes):
Recordit is not working on my machine as of now, so I am providing screenshots for quicreferencece. 
image
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/HomePage/index.js [hunk 1]
```diff
@@ -11,16 +11,38 @@ import ListItemText from "@material-ui/core/ListItemText";
 import List from "@material-ui/core/List";
 import { userList } from "./userList";
 import useStyles from "./styles";
+import SideBar from "../SideBar/index";
 
 function HomePage({ background = "white", textColor = "black" }) {
   const classes = useStyles();
-  const [value, setValue] = useState(2);
+    const [value, setValue] = useState(2);
+    const [footerContent, setFooterContent] = useState([
```
Comment (Niweera) [inline src/components/HomePage/index.js:19]: is this component gets used in many places?

Comment (Janvi-Thakkar) [reply src/components/HomePage/index.js:19]: Not as of now. But considering the possibility of using floating links, I have tried to make it dynamic.

Comment (Niweera) [reply src/components/HomePage/index.js:19]: so shall we export the entire useEffect to another standalone component?

https://reactjs.org/docs/hooks-custom.html

**Comment by Janvi-Thakkar** [comment]
@Niweera Can you please review and merge this?

**Comment by Niweera** [comment]
Can you resolve the merge conflicts?

**Comment by Janvi-Thakkar** [comment]
> Can you resolve the merge conflicts?

Resolved

**Pull request merged**